File: Source/frontend/frontend.Service/IStabilisationsPerCargoService.cs
using frontend.Domain;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace frontend.Service
{
    public interface IStabilisationsPerCargoService
    {
        List<StabilisationPerCargo> All();
        StabilisationPerCargo Find(int id);
        void Add(StabilisationPerCargo stabilisationsPerCargo);
        void Update(StabilisationPerCargo stabilisationsPerCargo);
        void Delete(int id);
    }
}

File: source/back-end/src/main/java/be/pxl/backend/service/IStabilisationsPerCargoService.java
package be.pxl.backend.service;

import java.util.List;

import be.pxl.backend.entity.StabilisationPerCargo;

public interface IStabilisationsPerCargoService {
	
	public StabilisationPerCargo find(int id);
	
	public List<StabilisationPerCargo> all();
	
	public void persist(StabilisationPerCargo stabilisationPerCargo);
	
	public void delete(int id);
	
}
